src/resolvers/index.ts

```diff
@@ -22,10 +23,27 @@ const noPerms = {
 
 export const resolvers: Resolvers<Context> = {
   Query: {
-    me: (_, _params, ctx) => {
+    me: async (_, _params, ctx) => {
+      let username;
       if (!ctx.session.user) return null;
-      console.log(ctx.session.user.username);
-      return null;
+      else if (!ctx.session.user.username) return null;
+      else username = ctx.session.user.username;
+
+      return await getMe(username);
+    },
+    findUserById: async (_, _params, ctx) => {
+      if (!ctx.session.user) return null;
+      else if (!_params.id) return null;
+      else return await getUserByID(+_params.id);
```

Comment:
Don't need the `else` here either. Also you can remove the `_` from params now since the value is being used.